src/coordinated_workers/coordinator.py

```diff
@@ -807,31 +831,42 @@ def _render_workers_alert_rules(self):
                 "charm_name": worker_topology["charm_name"],
             }
             topology = cosl.JujuTopology.from_dict(topology_dict)
-            alert_rules = cosl.AlertRules(query_type="promql", topology=topology)
-            alert_rules.add_path(WORKER_ORIGINAL_ALERT_RULES_PATH, recursive=True)
-            alert_rules_contents = yaml.dump(alert_rules.as_dict())
 
-            file_name = (
-                f"{CONSOLIDATED_ALERT_RULES_PATH}/rendered_{worker_topology['application']}.rules"
-            )
-            with open(file_name, "w") as writer:
-                writer.write(alert_rules_contents)
+            for orig_path, consolidated_path, type in alert_rules_sources:
+                alert_rules = cosl.AlertRules(query_type=type, topology=topology)  # type: ignore
+                alert_rules.add_path(orig_path)
+                alert_rules_contents = yaml.dump(alert_rules.as_dict())
 
-    def _remove_rendered_alert_rules(self):
-        files = glob.glob(f"{CONSOLIDATED_ALERT_RULES_PATH}/rendered_*")
+                file_name = f"{consolidated_path}/rendered_{worker_topology['application']}.rules"
+                with open(file_name, "w") as writer:
+                    writer.write(alert_rules_contents)
+
+    def _remove_rendered_alert_rules(self, consolidated_path: str):
+        files = glob.glob(f"{consolidated_path}/rendered_*")
         for f in files:
             os.remove(f)
 
-    def _consolidate_nginx_alert_rules(self):
+    def _render_nginx_alert_rules(self):
         """Copy Nginx alert rules to the consolidated alert folder."""
-        for filename in glob.glob(os.path.join(NGINX_ORIGINAL_ALERT_RULES_PATH, "*.*")):
-            shutil.copy(filename, f"{CONSOLIDATED_ALERT_RULES_PATH}/")
+        alerts_paths = (
+            (NGINX_ORIGINAL_METRICS_ALERT_RULES_PATH, CONSOLIDATED_METRICS_ALERT_RULES_PATH),
+            (ORIGINAL_LOGS_ALERT_RULES_PATH, CONSOLIDATED_LOGS_ALERT_RULES_PATH),
+        )
+
+        for orig_path, consolidated_path in alerts_paths:
+            for filename in glob.glob(os.path.join(orig_path, "*.*")):
+                shutil.copy(filename, f"{consolidated_path}/")
 
     def _render_alert_rules(self):
         """Render the alert rules for Nginx and the connected workers."""
-        os.makedirs(CONSOLIDATED_ALERT_RULES_PATH, exist_ok=True)
+        for consolidated_path in (
+            CONSOLIDATED_METRICS_ALERT_RULES_PATH,
+            CONSOLIDATED_LOGS_ALERT_RULES_PATH,
+        ):
+            os.makedirs(consolidated_path, exist_ok=True)
+            self._remove_rendered_alert_rules(consolidated_path)
         self._render_workers_alert_rules()
-        self._consolidate_nginx_alert_rules()
+        self._render_nginx_alert_rules()
```

Comment:
given that the globals are called 'consolidated', why is this 'render' and not 'consolidate'? or should the globals be 'rendered'?

Reply:
sure, and the mutation that is happening we're apparently calling 'consolidation'.

How about 'merge'?
- MERGED_ALERT_RULES
- src/alert_rules/merged_something.rule
- `self._merge_nginx_alert_rules()`